components/provider/src/error.rs

```diff
@@ -52,13 +52,13 @@ pub enum Error {
 
     /// The data provider encountered some other error when loading the resource, such as I/O.
     #[error("Failed to load resource: {0}")]
-    Resource(#[from] Box<dyn std::error::Error>),
+    Resource(#[from] Box<dyn std::error::Error + Send + Sync>),
```

Comment:
Here and elsewhere: I needed to add `+ Send + Sync` to make these errors compatible with `anyhow`.  Since external users may also be using `anyhow`, this seems like a positive change.